src/sentry/static/sentry/app/types/events.ts

```diff
@@ -1,7 +1,10 @@
-export interface Thread {
-  id: string;
+import {StacktraceType, RawStacktrace} from './stacktrace';
+
+export interface ThreadType {
```

Comment:
I thought we were trying to avoid using `Type` in the type names, and instead chose non-conflicting names for display components.